packages/@o3r/core/README.md

```diff
@@ -76,7 +76,7 @@ Otter framework provides a set of code generators based on [angular schematics](
 | store-simple-sync          | Create a simple sync new store in your library / application.                 | `ng g store-simple-sync`          |
 | store-action               | Create an action into an existing store.                                      | `ng g store-action`               |
 | renovate-bot               | Create a basic Renovate Bot                                                   | `ng g renovate-bot`               |
-| module                     | Create a new Otter compatible module in your monorepo                         | `ng g module`                     |
+| library                    | Add a new Otter Module to the current project                                 | `ng g library`                    |
```

Comment:
IMO this PR should target `9.0`. Otherwise we will be with `module` in 9.0 and `library` in 9.1.
And it's more a fix then a feature.